src/Symfony/Bridge/Doctrine/Form/ChoiceList/DoctrineChoiceLoader.php

```diff
@@ -93,9 +98,7 @@ public function loadChoiceList($value = null)
             ? $this->objectLoader->getEntities()
             : $this->manager->getRepository($this->class)->findAll();
 
-        $this->choiceList = $this->factory->createListFromChoices($objects, $value);
-
-        return $this->choiceList;
+        return $this->choiceList = new ArrayChoiceList($objects, $value);
```

Comment:
This is broken, as it means that ``$value`` cannot be a property path anymore (as you don't handle the conversion to a closure anymore)